ts/Extensions/ParallelCoordinates.ts

```diff
@@ -678,19 +679,28 @@ namespace ParallelAxis {
         }
 
         if (chart && chart.hasParallelCoordinates && !axis.isXAxis) {
-            const index = parallelCoordinates.position,
-                currentPoints: Array<Point> = [];
+            const index = parallelCoordinates.position;
+            let currentPoints: Array<number|null> = [];
 
             axis.series.forEach(function (series): void {
                 if (
+                    series.yData &&
                     series.visible &&
-                    defined((series.yData as any)[index as any])
+                    isNumber(index)
                 ) {
-                    // We need to use push() beacause of null points
-                    currentPoints.push((series.yData as any)[index as any]);
+                    const y = (series.yData)[index];
+
+                    // Take into account range series points as well (#15752)
+                    if (isArray(y)) {
+                        currentPoints = currentPoints.concat(y);
+                    } else {
+                        currentPoints.push(y);
+                    }
                 }
             });
 
+            currentPoints = currentPoints.filter((y): boolean => defined(y));
```

Comment:
Can probably be simplified to 

`currentPoints = currentPoints.filter(defined);`

Or, even safer and probably as efficient:

`currentPoints = currentPoints.filter(isNumber);`

Reply:
Sure, thanks for all the feedback.